packages/shared/test-utils/src/database.ts

```
@@ -109,3 +69,59 @@ export async function teardownTestDatabase() {
+    })
+
+  } catch (error) {
+    throw new Error(`Database cleanup transaction failed: ${error}`)
+  }
+
+  // Report errors but don't fail the cleanup unless all tables failed
+  if (errors.length > 0) {
+    const errorMessage = `Failed to truncate ${errors.length} table(s):\n` +
+      errors.map(({ table, error }) => `  - ${table}: ${error}`).join('\n')
+    
+    if (errors.length === tables?.length) {
```

ESLint

[error] 'tables' is not defined.

(no-undef)